.github/workflows/pr-check.yml

```diff
@@ -26,6 +26,18 @@ jobs:
         with:
           version: v1.54.2
           skip-pkg-cache: true
+      - name: Fix License Header
+        uses: apache/skywalking-eyes/header@main
+        with:
+         mode: fix
+      - name: Apply Changes
+        uses: EndBug/add-and-commit@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          author_name: License Bot
+          author_email: license_bot@github.com
+          message: 'Automatic application of license header'
```

Comment:
```suggestion
      - name: Fix License Header
        uses: apache/skywalking-eyes/header@main
        with:
         mode: fix
      - name: Apply Changes
        uses: EndBug/add-and-commit@v4
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          author_name: License Bot
          author_email: license_bot@github.com
          message: 'Automatic application of license header'
```

I see that this actually setups Go 1.18, which hides our setup of Go 1.20.
I think that if we move this before setup-go step it may work.

Reply:
fixed